data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -1013,15 +1002,9 @@
     "        temp_df[index.priority_communities_field] == True\n",
     "    )\n",
     "\n",
-    "    grouped_df = (\n",
-    "        temp_df.groupby(index.priority_communities_field).mean().reset_index()\n",
-    "    )\n",
-    "    result_df = grouped_df[\n",
-    "        [index.priority_communities_field] + comparison_fields\n",
-    "    ]\n",
-    "    result_df.to_csv(\n",
-    "        directory / f\"{index.method_name} Basic Stats.csv\", index=False\n",
-    "    )\n",
+    "    grouped_df = temp_df.groupby(index.priority_communities_field).mean().reset_index()\n",
```

Comment:
With all these black reformattings, I think (but I'm not 100% sure) there's a slight difference when you run `black .` from the command line on these files and when you run `black` inside of the notebook. My guess is that it's due to the additional quotes and linebreak characters causing the line limit to be exceeded when you look at the code from outside the notebook. 

Maybe we create a backlog ticket to investigate this and clean this up (e.g., exclude the notebooks from black when running from command line) so we don't keep shifting the code back and forth rather pointlessly.

Reply:
@emma-nechamkin it's easy to set "Format on Save" on VSCode and that would use Black. [Here's some instructions.](https://marcobelo.medium.com/setting-up-python-black-on-visual-studio-code-5318eba4cd00).